src/Microsoft.ML.Transforms/Text/WordEmbeddingsExtractor.cs

```diff
@@ -629,7 +629,7 @@ private string EnsureModelFile(IHostEnvironment env, out int linesToSkip, WordEm
                     string dir = kind == WordEmbeddingEstimator.PretrainedModelKind.SentimentSpecificWordEmbedding ? Path.Combine("Text", "Sswe") : "WordVectors";
                     var url = $"{dir}/{modelFileName}";
                     var ensureModel = ResourceManagerUtils.Instance.EnsureResourceAsync(env, ch, url, modelFileName, dir, Timeout);
-                    ensureModel.Wait();
+                    ensureModel.Wait(5 * 60 * 1000); // 5 minute timeout period
```

Comment:
Too low for the 6GB fastText word embedding.

This would require a 164 Mbps connection to finish in 5 min.

Reply:
As a baseline, this would timeout for me on my home connection: _(and for a connection 4x faster than mine)_
![image](https://user-images.githubusercontent.com/4080826/76811633-8108d980-67af-11ea-9e9b-f7ddf4d28304.png)

I expect I'll see similar results, but I'm checking the download time of the fastText 6GB model on my connection. https://aka.ms/tlc-resources/WordVectors/wiki.en.vec.

Reply:
Thanks for bringing this up @justinormont . I made this edit to make sure these tests would not hang in our CI builds, and it seems they have made no difference.